src/main/frontend/src/components/App.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Navbar from './Navbar.js';
+import WordShow from './WordContainer.js';
+import WordShow from './WordShow.js';
```

Comment:
Given that you don't yet have a WordContainer or WordShow component, merging this would involve merging broken code into `main`, which is not ideal.

Two solutions:
1. Remove lines 4-5 and 13-18 until you're ready to build those components. _This is the best practice, to keep those components for later so that they show up as completely new files in the appropriate PRs._
2. Create empty components that simply render a header. Not ideal, since it muddies the lines between PRs.

Please fix this up and re-post as a new request for review once done!

Reply:
Removed those lines